packages/edge-login-ui-rn/src/components/themed/SimpleSceneHeader.js

```diff
@@ -0,0 +1,48 @@
+// @flow
+
+import * as React from 'react'
+import { View } from 'react-native'
+import { cacheStyles } from 'react-native-patina'
+
+import {
+  type Theme,
+  type ThemeProps,
+  withTheme
+} from '../services/ThemeContext'
+import { EdgeText } from '../themed/EdgeText'
+
+type Props = {
+  children: React.Node
+}
+
+const SimpleSceneHeaderComponent = ({
+  children,
+  theme
+}: Props & ThemeProps) => {
+  const styles = getStyles(theme)
+
+  return (
```

Comment:
You can use the "Divider" component from the previous comment instead of having it as part of the styling.